Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -139,6 +143,15 @@ class XMLHttpRequestBase {
     }
   }
 
+  _stringToArrayBuffer(str: string): ArrayBuffer {
+    var buffer = new ArrayBuffer(str.length);
+    var bufferView = new Uint8Array(buffer);
```

Comment:
@sreesharp It'll probably create confusion. Can you create another PR only adding reponseText and responseType? Sorry for the trouble.